From 6b93a151584dff70141198462a1d08cdc2f540e6 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 18 Jul 2017 15:01:31 +0800
Subject: [PATCH 2152/5242] MLK-16015-3 arm64: fsl-imx8qm.dtsi: Add ldb
 aliases

commit  55da2c88aea1c1c5c2f73f5e7866ea53ba703a56 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds ldb aliases so that the relevant driver is
able to distinguish between the two LDB instances.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 6eb7449..d42a775 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -36,6 +36,8 @@
 		dpu1 = &dpu2;
 		ethernet0 = &fec1;
 		ethernet1 = &fec2;
+		ldb0 = &ldb1;
+		ldb1 = &ldb2;
 		isi0 = &isi_0;
 		isi1 = &isi_1;
 		isi2 = &isi_2;
-- 
1.7.9.5

